profiler/lib.profiler/nbproject/project.properties

```diff
@@ -45,6 +45,6 @@ nbm.executable.files=\
     lib/deployed/**/hpux*/lib*.sl,\
     remote-pack-defs/*.sh
 
-javac.source=1.7
-javac.target=1.7
-bootclasspath.prepend=${build.dir}/disable-release
\ No newline at end of file
+javac.source=1.8
+javac.target=8
```

Comment:
small nit pick. this should probably be 1.8 for consistency reasons and to make it easier to find.

Reply:
I've got a few more similar patches - will sort them all out at once.

Reply:
we have some cleanup patches pending. e.g https://github.com/apache/netbeans/pull/3278 and i think i saw another one, can't find it right now. Just as heads up regarding conflicts in those files.